app/AuthForm.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+import FormInput from "@/components/FormInput";
+import styles from "./page.module.css";
+import { SyntheticEvent } from "react";
+import Link from "next/link";
+
+const handleSubmit = async (e: SyntheticEvent) => {
+  e.preventDefault();
+  const target = e.target as typeof e.target & {
+    email: { value: string };
+    password: { value: string };
+  };
+
+  const body = JSON.stringify({
+    email: target.email.value,
+    password: target.password.value,
+  });
+
+  const res = await fetch("http://localhost:8080/auth/login", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: body,
+  });
+  const data = await res.json();
```

Comment:
Can we use axios for this? We'll also need to not hard-code this URL.

Reply:
Listo